test/Microsoft.AspNetCore.Razor.Runtime.Test/Microsoft.AspNetCore.Razor.Runtime.Test.csproj

```diff
@@ -7,10 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="**\*.cs" />
+    <Compile Include="**\*.cs" Exclude="TestFiles\**\*" />
     <Compile Include="..\..\shared\Microsoft.AspNetCore.Razor.TagHelpers.Testing.Sources\**\*.cs" />
-    <Content Include="TestFiles\**\*" CopyToOutputDirectory="Always" />
-    <EmbeddedResource Include="TestFiles\**\*" />
```

Comment:
This is supposed to be here, I don't know how this could be passing without it....